sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/aggregation/MaxAggregator.java

```diff
@@ -13,8 +19,47 @@ public MaxAggregator() {
         this.value = Undefined.value();
     }
 
+    static Object getValue(Object value) {
+
+        if (value instanceof TextNode) {
+            return ((JsonNode) value).asText();
+        } else if (value instanceof NumericNode) {
+            return ((JsonNode) value).numberValue();
+        } else if (value instanceof BooleanNode) {
+            return ((JsonNode) value).asBoolean();
+        } else if (value instanceof NullNode) {
+            return null;
+        } else {
+            return value;
+        }
+    }
+
     @Override
     public void aggregate(Object item) {
+
+        if (item instanceof ObjectNode) {
+            ObjectNode objectNode = (ObjectNode) item;
+            if (objectNode.hasNonNull("count")) {
```

Comment:
we should use constant for all of these "count", "max" etc, here and other places.

Reply:
done